### ISSUE TITLE
Hand Rotation problem 

### ISSUE THREAD

**Issue by liudongdong1** [caizhongang/SMPLer-X#52]
Hi, thanks for your great work again.
I am utilizing the SMPL-X-H32 model to process an online video featuring extensive hand movements. When attempting to transform the model's hand rotation output into BVH motion files, I've observed that the resulting hand rotations are less than ideal. Despite adjusting the orientations, which improved some frames, inconsistencies persist in the hand orientations across others.

question 1. The degree of finger bending is too slight， and for some frame, the direction of finger bending seems wrong;
![image](https://github.com/caizhongang/SMPLer-X/assets/25970295/df287c11-8c62-4df2-b497-977c3f27bc3f)
![image](https://github.com/caizhongang/SMPLer-X/assets/25970295/d8764be7-e7ae-43db-8a73-1a409432f6a7)
question 2. i modify the direction of the above frame, it seems ok, but other frames seem wrong;
![image](https://github.com/caizhongang/SMPLer-X/assets/25970295/89ddf5d6-1f04-49d3-b61a-83e5de22398a)
![image](https://github.com/caizhongang/SMPLer-X/assets/25970295/c201d05e-abd6-4441-8f83-abd0bc4229b6)
question3. The facial parameters seem not taken effect.
`class FaceRegressor(nn.Module):
    def __init__(self, feat_dim=768):
        super(FaceRegressor, self).__init__()
        self.expr_out = make_linear_layers([feat_dim, smpl_x.expr_code_dim], relu_final=False)
        self.jaw_pose_out = make_linear_layers([feat_dim, 6], relu_final=False)

    def forward(self, expr_token, jaw_pose_token):
        expr_param = self.expr_out(expr_token)  # expression parameter
        jaw_pose = self.jaw_pose_out(jaw_pose_token)  # jaw pose parameter
        return expr_param, jaw_pose`

**Comment by zxx123518** [comment]
Hello, I am also studying hand Rotation problem at present, could you share the code related to your movement output? Thank you very much！

**Comment by WingkitChou** [comment]
model hand's initial pose is different, use relaxing hand instead.

**Comment by Wei-Chen-hub** [comment]
Hi, the problem seems related to `flat_hand_mean`, smplerx assumes `'flat_hand_mean'=False`. Please check this kwarg when creating the smplx model.

**Comment by TDIT-haha** [comment]
I want to know if this problem was solved later? What is the reason?

**Comment by TDIT-haha** [comment]
> 我想知道这个问题后来解决了吗？是什么原因？

后来我解决了问题，可以参考https://github.com/IDEA-Research/OSX/issues/84的问题解决方案，简单的解决方案是我在推理输出中将smplx的full_pose提取出来，这是有经过偏移处理的，再转fbx文件后手就没有问题了

**Comment by iamthephd** [comment]
> > 我想知道这个问题后来解决了吗？是什么原因？
> 
> 后来我解决了问题，可以参考[https://github.com/IDEA-Research/OSX/issues/84的问题解决方案，简单的解决方案是我在推理输出中将smplx的full_pose提取出来，这是有经过偏移处理的，再转fbx文件后手就没有问题了](https://github.com/IDEA-Research/OSX/issues/84%E7%9A%84%E9%97%AE%E9%A2%98%E8%A7%A3%E5%86%B3%E6%96%B9%E6%A1%88%EF%BC%8C%E7%AE%80%E5%8D%95%E7%9A%84%E8%A7%A3%E5%86%B3%E6%96%B9%E6%A1%88%E6%98%AF%E6%88%91%E5%9C%A8%E6%8E%A8%E7%90%86%E8%BE%93%E5%87%BA%E4%B8%AD%E5%B0%86smplx%E7%9A%84full_pose%E6%8F%90%E5%8F%96%E5%87%BA%E6%9D%A5%EF%BC%8C%E8%BF%99%E6%98%AF%E6%9C%89%E7%BB%8F%E8%BF%87%E5%81%8F%E7%A7%BB%E5%A4%84%E7%90%86%E7%9A%84%EF%BC%8C%E5%86%8D%E8%BD%ACfbx%E6%96%87%E4%BB%B6%E5%90%8E%E6%89%8B%E5%B0%B1%E6%B2%A1%E6%9C%89%E9%97%AE%E9%A2%98%E4%BA%86)

Link not working!

**Comment by sonukiller** [comment]
> I want to know if this problem was solved later? What is the reason?

Yes, I was able to solve the problem using `flat_hand_mean=False` in kwarg while creating SMPLX model.
